code/game/objects/items/devices/scanners.dm

```diff
@@ -693,12 +693,82 @@ GENE SCANNER
 		else
 			to_chat(user, "Possible mutations: [T.slime_mutation[1]], [T.slime_mutation[2]], [T.slime_mutation[3]], [T.slime_mutation[4]]")
 			to_chat(user, "Genetic destability: [T.mutation_chance] % chance of mutation on splitting")
-	if (T.cores > 1)
+	if(T.cores > 1)
 		to_chat(user, "Multiple cores detected")
 	to_chat(user, "Growth progress: [T.amount_grown]/[SLIME_EVOLUTION_THRESHOLD]")
 	if(T.effectmod)
 		to_chat(user, "<span class='notice'>Core mutation in progress: [T.effectmod]</span>")
 		to_chat(user, "<span class = 'notice'>Progress in core mutation: [T.applied] / [SLIME_EXTRACT_CROSSING_REQUIRED]</span>")
+	if(T.transformeffects != SLIME_EFFECT_DEFAULT)
+		var/slimeeffects = "\nTransformative extract effects detected: "
+		//var/i = T.effectsapplied - 1
+		var/i = 0
```

Comment:
There has got to be a better way to do it than this, seems kinda shitcodey

Reply:
I know I just kept this here in case in the future someone wanted to make affections stackable. I will change it.

Reply:
ready.